packages/bpk-component-swap-button/src/BpkSwapButton.module.scss

```diff
@@ -0,0 +1,79 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2016 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+@import '../../bpk-mixins/index.scss';
+
+@layer bpk-component {
+  .bpk-swap-button {
+    position: absolute;
+    top: 50%;
+    z-index: 1;
+    width: bpk-spacing-xxl();
+    height: bpk-spacing-xxl();
+    transform: translateY(-50%) rotate(90deg);
+
+    // half the buttons width plus half the gap between origin and destination buttons
+    margin-inline-start: -1 * ((bpk-spacing-xxl() / 2) + (bpk-spacing-sm() / 2));
```

Comment:
I wonder if this styling should be left to the consumer. We may have other use cases of this button that don't need to do this.

The button being exported from backpack shouldn't dictate positioning. 

For our use case to position it we would render the button inside a container that sets these values

Reply:
That makes sense, I got a bit lost at what styling should be in bpk vs banana

Reply:
@LouiseReid would it also make sense to leave 
```
    position: absolute;
    top: 50%;
    z-index: 1;
```
for implementation stage as well?

Reply:
Yea they should be left out too